docs/user_manual/european_nucleotide_archive_support.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: page
+title: "24. European Nucleotide Archive Support"
+section: 24
+---
+
+
+Once sequencing is complete and analysis has been performed, the generated data
+may be deposited in an archive, such as the [ENA](https://www.ebi.ac.uk/ena)
+(formerly ERA; previously connected to the SRA). The archive requires not only
+the sequencing data, but the biological metadata associated with the data. To
+facilitate this process, MISO provides a mechanism to export the appropriate
+metadata required. This requires adding some extra information to MISO during
+the sequencing process.
+
+
+{% assign sub = 1 %}
+{% include userman-heading.md section=section sub=sub section=page.section title="Studies" %}
+
+[A study](https://ena-docs.readthedocs.io/en/latest/prog_02.html#the-study-object)
+is a unit of work to be submitted to the ENA in a batch. A study in MISO is
+limited in scope to a project. Although conceptually similar to a subproject,
+studies exist in MISO solely for ENA support.
+
+{% assign subsub = 1 %}
+{% include userman-heading.md sub=sub subsub=subsub section=page.section title="Studies List" %}
+
+The studies list can be accessed by clicking the "Studies" link from the Tracking list at the left side of the
+screen. The page shows all of the studies from all projects. At the top of the
+table, you will find controls for adding, and deleting studies.
+
+You can click on a study name or alias to get to the Edit Study page.
+
+For a particular project, the studies can be found on the [project
+page](projects.html#edit_project_page). Click the "My Projects" tab near the
```

Comment:
* "[Edit Project page]"
* This section should not contain instructions to get there (My Projects tab, etc)